src/css/style.css

```diff
@@ -1225,20 +1235,103 @@ Detail page css end here
 }
 
 .button:hover,
-.button:active{
+.button:active {
   background: linear-gradient(to left, rgb(255, 255, 255), rgb(218, 218, 218), rgb(197, 197, 197), rgb(146, 146, 146));
-  box-shadow: 0px 5px 100px 10px #ffffff;;
+  box-shadow: 0px 5px 100px 10px #ffffff;
+  ;
 }
 
-.loginlink{
-  position:absolute;
+.loginlink {
+  position: absolute;
   margin-top: 35px;
-  color:rgb(22, 137, 101);
-  float:left;
+  color: rgb(22, 137, 101);
+  float: left;
 }
 
 .register_button:hover,
-.register_button:active{
+.register_button:active {
   background: linear-gradient(to left, rgb(255, 255, 255), rgb(218, 218, 218), rgb(197, 197, 197), rgb(146, 146, 146));
   box-shadow: 0px 5px 100px 10px #ffffff;
+}
+
+/*-----------------------login page css start----------------------------------*/
+
+.login {
+  background-color: rgb(46, 11, 102);
+  width: 450px;
+  height: 400px;
+  position: relative;
+  border-radius: 7%;
+  margin: auto;
+  border: top left radius 0%;
+}
+
+.login_form {
+  display: flex;
+  margin: auto;
+  background-color: rgb(122, 119, 226);
+  justify-content: center;
+  width: 400px;
+  height: 350px;
+  align-items: center;
+  padding: 20px;
+  border-radius: 7%;
+  position: absolute;
+  top: 15px;
+  right: 14px;
+  border-top-left-radius: 0%;
+  background: linear-gradient(to bottom, #00ABBE, #2057A9, #0A2E52, rgb(6059, 74));
+  box-shadow: 0px 60px 300px 35px #000000;
+}
+
+/*body {
+  font-family: 'Arima';
+  font-size: 20px;
+  margin: 30px;
+
+}*/
+
+label,
```

Comment:
need to add .login for all labels, buttons, input in css otherwise it will take over on all the other pages also.

Reply:
edited as per suggestion